Correcciones del populate

angbongon opened the pull request:
Se han corregido incoherencias en el seeding, como: asistentes con más de un enrollment aceptado, mayor número de asistentes que aforo del evento, usuarios que no superan la edad mínima como asistentes a eventos. Además de perfilar cosas como: las horas de inicio y fin de los eventos, hacer que no todos lo asistentes hagan rating sobre el host del evento y viceversa.

angbongon commented:
@juanmiguelruiz esperando a tu review

angbongon commented:
@raunavcru la petición POST se realiza correctamente pero sigue sin crear el objetco en BBDD

raunavcru commented:
Dime los datos para replicar el error

juanmiguelruiz commented:
> Dime los datos para replicar el error

estando el brach dev, hazte un usuario (superuser o no) e intenta crear un evento.

raunavcru commented:
El cambio no esta en la rama dev hasta que este aprobada la pull request.
Hay no estan los cambio que yo he hecho,
En mi rama funciona

angbongon commented:
No aparecen los botones para avanzar en el formulario

Commit:
[fix][WIP] Coherencia en ratings y enrollments

También se han generado horas más coherentes para los eventos.

Commit:
[style] Se método aux al final del archivo

Commit:
[fix] Mayor coherencia y aleatoriedad en seeding

Ahora no se añaden más asistentes a un evento de lo que permite su
aforo, al igual ocurre con la edad mínima. También se ha hecho que no
todos los asistentes hagan rating sobre el host del evento y viceversa.

Commit:
[refactor] Eliminada duplicación de código en seed

Se ha metido en la cte TIMEZONE el valor '+0000' que se usaba en varias
líneas, con objeto de evitar duplicación.
